pkg/metaserver/agent/metric/malachite/fetcher.go

```diff
@@ -474,8 +474,8 @@ func (m *MalachiteMetricsFetcher) processSystemCPUComputeData(systemComputeData
 			klog.Errorf("[malachite] parse cpu name %v with err: %v", cpu.Name, err)
 			continue
 		}
-		m.metricStore.SetCPUMetric(cpuID, consts.MetricCPUUsage,
-			utilmetric.MetricData{Value: cpu.CPUUsage, Time: &updateTime})
+		m.metricStore.SetCPUMetric(cpuID, consts.MetricCPUUsageRatio,
```

Comment:
maybe it's better to keep both those two metrics, since we will depend on the previous metric in other cases

Reply:
CPUUsage will be changed to CPUUsageRatio globally in all cases